lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -130,14 +130,6 @@ def script
 }
 #{Pod::Generator::ScriptPhaseConstants::INSTALL_DSYM_METHOD}
 #{Pod::Generator::ScriptPhaseConstants::STRIP_INVALID_ARCHITECTURES_METHOD}
-# Copies the bcsymbolmap files of a vendored framework
```

Comment:
@amorde line 67 above has `BCSYMBOLMAP_DIR="BCSymbolMaps"` is this used by xcframeworks?

Reply:
I am not sure if we need that anymore... but there is logic further below that scans fro bcsymbolmaps and installs them. Those also dont seem to be part of input/output paths?

Reply:
`BCSYMBOLMAP_DIR="BCSymbolMaps"` is still used

Reply:
I ended up keeping it so no changes here.